shaunkendrone/call-center-ticketing-system
# Ft get ticket by status

What does this PR do?
Allows agents to retrieve tickets based on their status.

Description of the task to be completed?
This PR specifies that agents are allowed to filter tickets based on their status. Once a status is chosen, the system retrieves and displays all tickets that match the selected status.

How should this be manually tested?
-Ensure that there are tickets available in the system with various status types (Open, InProgress, Resolved) to test the filtering functionality effectively.
-Choose a specific status from the available options
-Verify that the system retrieves and displays the tickets matching the selected status. The displayed tickets should be filtered according to the chosen status criteria.
-Test other cases of status by entering a non existing status or empty status.Invalid input messages should be displayed.

Screenshots
![Screenshot 2023-07-08 001712](https://github.com/shaunkendrone/call-center-ticketing-system/assets/98257312/26fac283-91c9-495e-b91d-cc7938195ca2)
![Screenshot 2023-07-08 001738](https://github.com/shaunkendrone/call-center-ticketing-system/assets/98257312/ed9db5f2-87bc-42f3-9674-97ee2923bf0a)
